plugins/SliceInfoPlugin/SliceInfo.py

```diff
@@ -24,6 +21,34 @@
 
 catalog = i18nCatalog("cura")
 
+class SliceInfoJob(Job):
+    data = None
+    url = None
+
+    def __init__(self, url, data):
+        super.__init__(self)
+        self.url = url
+        self.data = data
+
+    def run(self):
+        if not self.url or not self.data:
+            Logger.log("e", "URL or DATA for sending slice info was not set!")
+            return
+
+        # Submit data
+        kwoptions = {"data" : self.data,
+                     "timeout" : 5
+                     }
+
+        if Platform.isOSX():
+            kwoptions["context"] = ssl._create_unverified_context()
+
+        try:
+            f = urllib.request.urlopen(self.url, **kwoptions)
+            Logger.log("i", "Sent anonymous slice info to %s", self.url)
+            f.close()
+        except Exception:
+            Logger.logException("e", "An exception occurred while trying to send slice information")
```

Comment:
Like with the update-checker, I would prefer to use Logger.log() instead of Logger.logException. There is not much use to dump the whole traceback to the log on every slice. Just a simple warning will do.